template.yaml

```diff
@@ -37,16 +45,24 @@ Resources:
       Runtime: python3.9
       Environment:
         Variables:
+          ApiChartOfAccounts: '/accounts'
+          ApiValidTags: '/tags'
+          CacheTTL: !Ref CacheTTL
           MipsOrg: !Ref MipsOrganization
           SsmPath: !Ref SsmParamPrefix
-          CacheTTL: !Ref CacheTTL
-          apiAllCostCenters: '/all/costcenters.json'
+          CodesToOmit: !Ref CodesToOmit
+          CodesToAdd: !Ref CodesToAdd
       Role: !GetAtt FunctionRole.Arn
       Events:
-        AllCostCenters:
+        ChartOfAccounts:
+          Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
+          Properties:
+            Path: '/accounts'
```

Comment:
please update the `Triggering` and `Response Format` section for updated endpoints

Reply:
done